lib/EarlyFunctionAnalysis/EarlyFunctionAnalysis.cpp

```diff
@@ -464,6 +475,60 @@ CFEPAnalyzer<FunctionOracle>::CFEPAnalyzer(llvm::Module &M,
   }
 }
 
+static void
+initializeCache(llvm::Module &M,
+                FunctionAnalysisResults &Properties,
+                const std::vector<llvm::GlobalVariable *> &ABIRegisters,
+                model::Binary &Binary) {
+  using namespace model;
+  using RegisterState = model::RegisterState::Values;
+
+  for (const auto &F : Binary.Functions) {
```

Comment:
Do not abbreviate ([B.6](https://pad.rev.ng/#B-CC)) and avoid `auto` ([B.15](https://pad.rev.ng/#B-CC)).